gammapy/modeling/parameter.py

```diff
@@ -138,6 +136,8 @@ def __init__(
             self.value = float(value)
             self.unit = unit
 
+        self.min = min
+        self.max = max
```

Comment:
To avoid having to check between `str` and `Quantity` in every place, you can convert the str to a quantity here

```
if isinstance(min, str):
      self.min = u.Quantity(min)
```

Reply:
I understand your argument but at the same time I think it's cleaner to put this in the setter. Moreover, one most probably set min and max after the `Parameter` initialisation.

Reply:
i agree. Then you can just simplify the setter by changing the str to a qty

Reply:
Done !